common/resources.yaml

```diff
@@ -0,0 +1,67 @@
+groups:
+ - data_type_def: ExternalResources
+   data_type_inc: Container
+   doc: A pair of tables for tracking external resource references in a file
+   datasets:
+    - data_type_inc: Data
+      name: keys
+      doc: A table for identifying which objects in a file contain
+      dtype:
+       - name: key_name
+         dtype: text
+         doc: The user term that maps to one or more resources in the 'resources' table.
```

Comment:
Am I reading this right that this is a complex data type of length 1? What is the advantage of that over just using a string dtype?

Reply:
The complex data type makes it easier to treat the dataset as a table.